Mathmagician/Fibonacci.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    class Fibonacci
+    {
+        public List<int> Make_Fibonacci(int numbersToAdd)
+        {
+            int x = numbersToAdd;
+            int a = 0;
+            int b = 1;
+            List<int> FibList = new List<int>();
+            for (int i = 0; i <= x; i++)
```

Comment:
Change the `<=` to just `<` in order to get the correct number of integers returned in the list.